test/only-english-comments.test.js

```diff
@@ -29,6 +29,12 @@ ruleTester.run('comments-only-in-english', rule, {
 				// normal comment 1
 				// normal comment 2
 			`
+		},
+		{
+			code: '/* */'
+		},
+		{
+			code: '// '
```

Comment:
Just empty comments are valid cases. But we need to check also that rule determines both `//` and `/* */` as code comments

Reply:
We need to test if //and /**/ supported to be commented?

Reply:
It is good to have